Userland/Libraries/LibCore/EventLoop.cpp

```diff
@@ -379,7 +379,7 @@ void EventLoop::spin_until(Function<bool()> goal_condition)
         pump();
 }
 
-void EventLoop::pump(WaitMode mode)
+int EventLoop::pump(WaitMode mode)
```

Comment:
We should use an unsigned type to represent a count? 

```suggestion
size_t EventLoop::pump(WaitMode mode)
```

Reply:
Fixed!